File: src/simple_test/simple_test.py
import time
import asyncio

async def async_print():
    for i in "ab":
        print(i)
        await asyncio.sleep(1)
    print("Hello")


async def run():
    result = asyncio.create_task(async_print())
    result2 = asyncio.create_task(async_print())
    await asyncio.sleep(1)
    print("World")
    resp = await result, await result2
    print("O")
    return resp
    
    
event_loop = asyncio.get_event_loop()
try:
    resp = event_loop.run_until_complete(run())
    print(resp)
finally:
    event_loop.close()

File: src/simple_test/simpleTest.js
function sleep(ms) {
    return new Promise((resolve, reject) => {
        setTimeout(resolve, ms)
    })
}

async function asyncPrint() {
    for(let i of ['a', 'b']) {
        console.log(i)
        await sleep(1000)
    }
    console.log("Hello")
}

async function run() {
    let result = asyncPrint()
    let result2 = asyncPrint()
    await sleep(1000)
    console.log("World")
    let resp = [await result, await result2]
    console.log("O")
    return resp
}

run()